frontend/src/App.tsx

```diff
@@ -1,36 +1,93 @@
-import React, {Component} from 'react';
+import React, { Component } from 'react';
+import styled from 'styled-components';
 import logo from './logo.svg';
-import './App.css';
- 
-class App extends Component {
- 
-    state:any = {};
- 
-    componentDidMount() {
-        setInterval(this.hello, 250);
+import LandingPage from './views/Landing';
+import fetchHello from './api/hello';
+
+const StyledApp = styled.div`
```

Comment:
I think you're right about App being a component so I moved that to the components folder. As for StyledApp, since it's applying styling to the App component I kept it in the same file kind of like how Spectrum does it [here](https://github.com/withspectrum/spectrum/blob/alpha/src/components/card/index.js). Although it'd also make sense to put it in a separate file like they do [here](https://github.com/withspectrum/spectrum/tree/alpha/src/components/button). Lmk if you have a preference on that.

Reply:
I think maintaining the same structure Spectrum has makes sense, so your change is good IMO.